openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -1064,7 +1074,7 @@ def update_plot(ii):
         print("Saving the animation movie as "+MovefileName+" in "+ODBdir+"->"+LoadCase+" folder")
         ani.save(Movfile, writer='ffmpeg')
 
-    plt.show()
+    # plt.show()
```

Comment:
We need plt.show() to display the animation.

Reply:
Corrected.